.circleci/config.yml

```diff
@@ -12,36 +12,37 @@ workflows:
           name: build-1.10.5-alpine3.10
           airflow_version: 1.10.5
           distribution_name: alpine3.10
-          docker_repo: astronomerinc/ap-airflow
+          docker_repo: astronomerio/ap-airflow
+          extra_args: "--build-arg VERSION=$(curl https://pip.astronomer.io/simple/astronomer-certified/latest-1.10.5.build)"
```

Comment:
Unfortunately, this will fail for 1.10.5 as we don't publish wheels for 1.10.5

Reply:
Although if we plan to maintain 1.10.5 for a longer-term we should cherry-pick the commits that releases AC wheels to our PyPI